### ISSUE TITLE
ha-card-border-radius not applied to media-control card

### ISSUE THREAD

**Issue by tomlut** [home-assistant/home-assistant-polymer#3081]
**Home Assistant release with the issue:**

0.91.2

**Last working Home Assistant release (if known):**

Nil

**UI (States or Lovelace UI?):**

Lovelace

**Browser and Operating System:**

Firefox 66.0.3 (64bit), Windows 10

**Description of problem:**

The `media-control` card corners are not rounded with the theme option `ha-card-border-radius` as per other cards.

**Javascript errors shown in the web inspector (if applicable):**
```

```

**Additional information:**

**Comment by iantrich** [comment]
the current card is not inside a ha-card. i'll be doing a rewrite soon though and will resolve this.

**Comment by tomlut** [comment]
I'm using the following style with card-mod at the moment so no rush:

```
    style: |
      ha-card {
        border: solid 2px var(--primary-color);
      }
```

**Comment by zsarnett** [comment]
Fixed via #4761
